### PR TITLE
[JIRA-1750] remove qpid dependency

### PR DESCRIPTION
**Pull request by traflm**

### CODE REVIEW

**Comment by traflm** [comment]
jenkins, test

**Comment by DaveBirdsall** [comment]
jenkins, add user

**Comment by DaveBirdsall** [comment]
Looks good to me. Thanks for doing this, Ming.

File: core/sql/sqlmxevents/logmxevent_sq.cpp [hunk 1]
```diff
@@ -49,7 +49,7 @@
 #endif
 #include "common/sql.error_event.pb.h"
 #include "common/sql.info_event.pb.h"
-#include "wrapper/amqpwrapper.h"
```
Comment (sandhyasun) [inline core/sql/sqlmxevents/logmxevent_sq.cpp]: This file could also be cleaned up/deleted. But lets do this separately after ensuring all the event logging has been ported to the current logmxevent_traf.cpp file completely.

**Pull request merged**